src/wago-connector/index.ts

```diff
@@ -16,7 +16,7 @@ import { Fish } from '@actyx/pond'
 // define default settings use during the local development process when
 // when the app is not running in an ActyxOS runtime
 const defaultSettings = {
-  name: 'Machine2',
+  name: 'Wago Connector',
```

Comment:
What's about the `AX_APP_SETTINGS`, isn't that a part that needs to be changed?

Reply:
I removed app settings for this example as we will not support them in v2.